lib/msal-common/src/network/INetworkModule.ts

```diff
@@ -39,7 +39,8 @@ export interface INetworkModule {
      */
     sendPostRequestAsync<T>(
         url: string,
-        options?: NetworkRequestOptions
+        options?: NetworkRequestOptions,
+        cancellationToken?: number
```

Comment:
Do we really need to add timeout to this interface? If we really do, can we add it as part of `options` ?

Btw, if this is a timeout, just call it timeout, not "token". CancellationToken has more functionality, timeout is just one of them.

Reply:
It seems to me that this was a bug where the timeout was missing from the post request definition in InNetworkModule. In httpClient, we allow the timeout (cancellationToken) to be optionally provided. @sameerag, can you chime in here? Was this originally supposed to be timeout functionality, or a cancellationToken?

Reply:
Let's not get bogged down in this @Robbie-Microsoft - just open a bug and we'll treat it separately. Let's not make changes to the existing interface if they are not needed.

Reply:
Well regardless, this change is needed for Managed Identity since we're exposing the timeout to the developer.